bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -2541,28 +2545,25 @@ ImageData newImageData(int zoom) {
 	protected ImageHandle newImageHandle(int zoom) {
 		currentZoom = zoom;
 		int gcStyle = drawer.getGcStyle();
-		Image image;
 		if ((gcStyle & SWT.TRANSPARENT) != 0) {
 			int scaledHeight = DPIUtil.scaleUp(height, zoom);
 			int scaledWidth = DPIUtil.scaleUp(width, zoom);
 			/* Create a 24 bit image data with alpha channel */
 			final ImageData resultData = new ImageData (scaledWidth, scaledHeight, 24, new PaletteData (0xFF, 0xFF00, 0xFF0000));
 			resultData.alphaData = new byte [scaledWidth * scaledHeight];
-			image = new Image(device, resultData, zoom);
+			init(resultData, zoom);
 		} else {
-			image = new Image(device, width, height);
-		}
-		GC gc = new GC(new DrawableWrapper(image, zoom), gcStyle);
+			createBaseHandle(zoom, width, height);
+		};
+		GC gc = new GC(new DrawableWrapper(Image.this, zoom), gcStyle);
```

Comment:
Can't we then remove the `DrawableWrapper` and just call?
```suggestion
		GC gc = new GC(Image.this, gcStyle);
```

Reply:
Seems like it would make sense to add a regression test for that scenario.